alica_tests/src/test/test_alica_engine_plan_parser.cpp

```diff
@@ -575,10 +575,9 @@ class MultiConfigFolderTest : public AlicaEngineTest
     std::vector<std::string> getTestFolderPaths() const override
     {
         std::string path;
-#if defined(PLANS)
-        path = std::string{PLANS} + "/src/test";
+#if defined(THIS_PACKAGE_DIR)
+        path = std::string{THIS_PACKAGE_DIR} + "/etc/";
```

Comment:
really nit: `std::string` is not necessary, and this was something I passed on my PR, we can remove it.

Reply:
Actually it is necessary, two `const char*` cannot be added together